CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+## [0.5.0] - 2020-04-02
+
+### Added
+- APIManager CRD 3scale 2.8
+- Metrics endpoint include 3scale operator and product version [#290](https://github.com/3scale/3scale-operator/pull/290)
+- Move system-smtp to secret [#280](https://github.com/3scale/3scale-operator/pull/280)
+- Support s3 apicompatible service [#302](https://github.com/3scale/3scale-operator/pull/302)
+- Add support for PodDisruptionBudget [#308](https://github.com/3scale/3scale-operator/pull/308)
+- Operator-sdk v0.15.2 Go version v1.13 [#331](https://github.com/3scale/3scale-operator/pull/331)
+
+## [0.4.0] - 2019-11-28
+
+### Added
+- APIManager CRD 3scale 2.7
```

Comment:
Maybe comment that 2.7 is the first version that can be upgraded to? (this is, upgrade from 2.6 to 2.7 was the first time we implemented upgrade functionality)

Reply:
good point. Added.